manifests/instance.pp

```diff
@@ -306,17 +303,17 @@
     "tech_ops = 'Tech Ops'",
   ],
   $shiftless_depts = undef,
-  $interest_list = [ 
-    "console        = 'Consoles'", 
-    "arcade         = 'Arcade'", 
+  $interest_list = [
+    "console        = 'Consoles'",
+    "arcade         = 'Arcade'",
     "lan            = 'PC Gaming'",
-    "music          = 'Music'", 
+    "music          = 'Music'",
```

Comment:
hm. why did all this unrelated stuff change whitespace junk... i swore it wasn't in my final diff... need to revert